docs/index.md

```diff
@@ -1,17 +1,78 @@
-# Welcome to MkDocs
+# Melange
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+## OCaml for JavaScript developers
 
-## Commands
+---
 
-* `mkdocs new [dir-name]` - Create a new project.
-* `mkdocs serve` - Start the live-reloading docs server.
-* `mkdocs build` - Build the documentation site.
-* `mkdocs -h` - Print help message and exit.
+Melange is a fork of the OCaml compiler that compiles to JavaScript. Melange
+strives to provide the best integration with both the OCaml and JavaScript
+ecosystems. To know more about it start by reading the [introductory
```

Comment:
Would this be a good opportunity to mention something like:

> This document is divided into the following sections:
> ...


It'd also leave space for adding more information later.

Reply:
fair. the current topbar looks like it has enough sections already.